Getting Type error when using the functions

Getting the following Type errors, when using any exported function from the package, for playwright (running `tsc`).

```bash
1669 export function nullSafe<T extends Function>(func: T): (...args: any) => ReturnType<T>;
                                                                                         ~
node_modules/@bigbinary/neeto-cist/index.d.ts:1672:54 - error TS2304: Cannot find name 'Draft'.

1672 export function modifyWithImmer<T>(modifier: (draft: Draft<T>) => void, data: T): T;
                                                          ~~~~~
node_modules/@bigbinary/neeto-cist/index.d.ts:1673:54 - error TS2304: Cannot find name 'Draft'.

1673 export function modifyWithImmer<T>(modifier: (draft: Draft<T>) => void): (data: T) => T;
```

Also, is it possible to remove the peerDependency on "neeto-commons-frontend"?

cc: @ajmaln

--- a/typeTemplates/index.d.ts
+++ b/typeTemplates/index.d.ts
@@ -1,3 +1,5 @@
+import { Draft } from "immer";
+
 export type Primitives = symbol | string | number | boolean | null | undefined;
 export type ObjectAndPrimitives = Primitives | object;
 type KeyType = string | number | symbol;
@@ -338,9 +340,9 @@ export function transformObjectDeep(
 export function truncate(string: string, length: number): string;
 export function _truncate(string: NilOr<string>, length: number): NilOr<string>;
 
-export function nullSafe<T extends Function>(
+export function nullSafe<T extends (...args: any) => any>(
   func: T
-): (...args: any) => ReturnType<T>;
+): (...args: Parameters<T>) => ReturnType<T>;
 
 export function isNotPresent(object: any): boolean;
 export function isPresent(object: any): boolean;

